app/views/invitations/new.html.haml

```diff
@@ -0,0 +1,21 @@
+%section.invite.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_for @invitation, class: "invite" do |f|
+        %header
+          %h1 Invite a friend to join MyFlix!
+        %fieldset
+          .form-group
+            .row
+              .col-sm-4
+                = f.text_field(:recipient_name, class: "form-control", label: "Friend's Name")
```

Comment:
with `bootstrap_form_for` this:

```html
         .form-group
           .row
             .col-sm-4
               = f.text_field(:recipient_name, class: "form-control", label: "Friend's Name")
```
could be just 

```html
  = f.text_field(:recipient_name, class: "form-control", label: "Friend's Name")
```
try to experiment with it and read docs. I also think the form should have `layout: :horizontal` option and `column_col: "col-sm-4"`.